src/tls_migration.erl

```diff
@@ -0,0 +1,67 @@
+%% TLS migration script (temporary)
+%%
+%% Run from the console as:
+%% > l(tls_migration). tls_migration:main(ok).
+
+-module(tls_migration).
+
+-export([main/1]).
+
+-include_lib("ex_uri/include/ex_uri.hrl").
+-include("logplex_drain.hrl").
+-include("logplex_logging.hrl").
+
+main(_) ->
```

Comment:
Would be niced to pass in some options here like `dry_run` so the entire script can be run without saving any changes